### ISSUE TITLE
[Bug]: Pest complains about missing .env file when run within a docker container

### ISSUE THREAD

**Issue by PatrickMessierGoC** [pestphp/pest#1028]
### What Happened

When running Pest tests inside a docker container that does NOT contain either a .env.testing or a .env file, we systematically get the following warning when the first 2 tests are executed : `→ file_get_contents(/usr/share/nginx/lp-opn/.env): Failed to open stream: No such file or directory `.  

![image](https://github.com/pestphp/pest/assets/45016166/7eaa615c-4c70-4738-8d61-2f84097a123a)

Environment variables are injected inside the container using docker compose env_file option. 

![image](https://github.com/pestphp/pest/assets/45016166/0b9eb31d-0456-4c34-bed3-5ad7f0e445a9)

Surprisingly, the rest of the test are running just fine. 

We don't have this behaviour when running equivalent tests with phpunit.

### How to Reproduce

Starting with an existing Laravel app with a Pest Test suite:
1. Create a .dockerignore file and add .env and .env.testing to it.
1. Build a docker container with the application code
1. Start the application container.
1. Execute the Laravel Pest Suite from within the docker container.  

### Sample Repository

https://dev.azure.com/CSPS-EFPC-UX/learning-platform/_git/lp-opn

### Pest Version

2.24.0

### PHP Version

8.1.26

### Operation System

Linux

**Comment by fabio-ivona** [comment]
Have you tried to load the env file in the volume?

**Comment by PatrickMessierGoC** [comment]
@fabio-ivona No. In fact, the question here is not about getting the env files in the container but rather about why Pest behaves this way when the environment variables are "injected" in the container rather than defined via an .env.testing file that _sounds_ like optional based on [Laravel Docs](https://laravel.com/docs/10.x/testing#the-env-testing-environment-file). We know the environment variables are somehow available in the container because all remaining tests runs successfully.

**Comment by nunomaduro** [comment]
sorry, i don't pest with docker, so i am unable to assist you.
